### PR TITLE
Feature/fw

### PR DESCRIPTION
**Pull request by RobertoAldea**
Change project configuration, test and hal sys

### CODE REVIEW

File: firmware/Sources/Test/test.c [hunk 1]
```diff
@@ -172,6 +172,7 @@ TEST_result_e CanMainTest(void){
 
 TEST_result_e HalMainTest(void){
 	TEST_result_e test_res = TEST_RESULT_SUCCESS;
+<<<<<<< Updated upstream
```
Comment (pastorpflores) [inline firmware/Sources/Test/test.c:175]: wtf. A merge is not well resolver here?

File: firmware/project_config/EPC_CONF/epc_conf.h [hunk 2]
```diff
@@ -61,9 +61,15 @@ extern int32_t EPC_CONF_Ls_Curr[2], EPC_CONF_Ls_Volt[2], EPC_CONF_Ls_Volt_Ext[2]
 //#define EPC_CONF_PWM_ENABLED
 #define EPC_CONF_GPIO_ENABLED
 //#define EPC_CONF_ADC_DMA_ENABLED
+<<<<<<< Updated upstream
```
Comment (pastorpflores) [inline firmware/project_config/EPC_CONF/epc_conf.h:64]: another issue with a merge here??

**Review by pastorpflores — CHANGES_REQUESTED** [review]

File: firmware/Sources/Test/test.c (hunk 1 above)
Comment (RobertoAldea) [reply firmware/Sources/Test/test.c]: Solved in last commit

File: firmware/project_config/EPC_CONF/epc_conf.h (hunk 2 above)
Comment (RobertoAldea) [reply firmware/project_config/EPC_CONF/epc_conf.h]: Solved in last commit

**Review by pastorpflores — APPROVED** [review]

**Comment by pastorpflores** [comment]
@mariuscrsn please resolve conflicts :)

**Pull request merged**